src/zinc.rs

```diff
@@ -187,9 +151,10 @@ fn finish(z: &mut Cursive) {
         rootpasswd,
         username,
         passwd,
+        ..Default::default()
     };
 
     z.pop_layer();
 
-    install::begin_install(selection);
+    Install::start(selection);
```

Comment:
This works but could be better if it was `selection.start()` or maybe `install.start()` if you rename the selection var

Reply:
i see, this is kinda cool tbh